### ISSUE TITLE
A player can enter a protected zone when in combat

### ISSUE THREAD

**Issue by Animationking11** [ChanceSD/PvPManager-Issues#162]
**Describe the bug**
You can enter protected areas when in combat

**To Reproduce**
1. Use an elytra and fireworks to forcefully enter a protected zone (more in the video)

**Expected behavior**
If both players (with combat tag) enter the protected zone they can pvp even if in the zone pvp is enabled, This would be fixed if you fixed forcefully entering with fireworks so that's why I am reporting the entering part.

**Plugin and Server version**
PvPManager: 3.9.4
Server: Paper get-Paper-"af3b377"

**Screenshots/Video**
Video of me preforming the bug: https://www.youtube.com/watch?v=1P02lkirA_U

**Comment by Animationking11** [comment]
> Being able to PvP while inside the zone is an option in config, it's called vulnerable, just set to false. About entering the zone though, this repository is for the issues with the premium version and I believe that issue doesn't happen with premium since it was improved quite a bit to prevent entering. And judging from your plugin version you seem to be using the free version, so just copy this issue report here if possible: https://github.com/ChanceSD/PvPManager Either way I will try to improve that feature a bit on the free version.

Still, you can get in a protected area when in combat. That's what I am reporting.

**Comment by ChanceSD** [comment]
Yes thank you for the report, I will improve that in the free version.  
What I meant is that's already fixed in the premium version and that you reported in the wrong place but that's fine
